### ISSUE TITLE
Commit style

### ISSUE THREAD

**Issue by AndreasPetter** [rxin/c-_rockstar#1]
I have some issues with your commit style ;). I recommend to squash the last 2800 commits. (Just kidding. It's clear that coding wasn't the goal for this project)

**Comment by rxin** [comment]
Yup - I think we should create a commit style guide and follow that.
